packages/verifiable-credential/schemas/VerifiableCredential.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "VerifiableCredential",
+  "description": "A [[VerifiableCredential]] is fact-checking review/attestation of claims made (or reported) in a [[ClaimedCredential]]",
+  "type": "object",
+  "properties": {
+    "@context": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "_context": {
+      "type": "string"
+    },
+    "type": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "_type": {
+      "type": "string"
+    },
+    "id": {
+      "$ref": "#/definitions/CeramicDocId"
+    },
+    "issuer": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "ethereumAddress": { "type": "string" }
+      },
+      "required": ["id"]
+    },
+    "credentialSubject": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "ethereumAddress": { "type": "string" }
+      },
+      "required": ["id"]
+    },
+    "credentialSchema": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "type": { "type": "string" },
+        "_type": { "type": "string" }
+      },
+      "required": ["id"]
+    },
+    "issuanceDate": {
+      "type": "string"
+    },
+    "expirationDate": {
+      "type": "string"
+    },
+    "proof": {
+      "type": "object"
+    },
+    "evidence": {
+      "type": "object"
+    },
+    "credentialStatus": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "type": { "type": "string" },
+        "_type": { "type": "string" }
+      },
+      "required": ["id"]
+    }
+  },
+  "required": ["id", "issuer", "credentialSubject", "credentialSchema", "issuanceDate"],
+  "additionalProperties": true,
```

Comment:
Can't this be `false` please? It would be more consistent across implementations.

Reply:
Updated the PR with a single model with 3 schemas and a single definition